### ISSUE TITLE
Support the --offline command line switch

### ISSUE THREAD

**Issue by matsev** [michel-kraemer/gradle-download-task#33]
Feature Request: 

The `gradle-download-task` should support the  [--offline](https://docs.gradle.org/current/userguide/dependency_management.html#sub:cache_offline) command line switch and should neither attempt to download a file, nor check if a newer version exists if the `onlyIfNewer` property is used. If the file exists (as a result of a download executed earlier), the task should exit successfully, else it should fail. 

It may not be feasible when downloading a directory, but it should be possible to implement when the `dest` is a single file as in the `downloadZipFile` task that was copied from the [examples](https://github.com/michel-kraemer/gradle-download-task/blob/master/README.md#examples) in the README.md.

```gradle
ask downloadZipFile(type: Download) {
    src 'https://github.com/michel-kraemer/gradle-download-task/archive/1.0.zip'
    dest new File(buildDir, '1.0.zip')
}
```
Dependency: 'de.undercouch:gradle-download-task:2.0.0

**Comment by matsev** [comment]
[StartParameter.isOffline()](https://docs.gradle.org/current/javadoc/org/gradle/StartParameter.html#isOffline()) might be useful:
> Specifies whether the build should be performed offline (ie without network access).

**Comment by michel-kraemer** [comment]
Interesting idea. I'll have a look at it.

**Comment by michel-kraemer** [comment]
Please let me know if the commit I just made meets your expectations. I can then release a new version if you like.

**Comment by matsev** [comment]
I have only verified the usage when the source is a single file (since that is the current project setup), see below. From my point of view it looks good.

- If I disable my network connection and execute `gradle build`, the build fails as expected because
> java.net.UnknownHostException:  http://some.file.to.download.zip

- If I instead execute `gradle build --offline` the build passes which is what I proposed would happen.

- If I delete the downloaded file and execute `gradle build --offline` the build fails as expected because the `gradle-download-task` cannot download the requested file.
> Unable to download http://some.file.to.download.zip in offline mode.

Thanks!
